.gitignore

```diff
@@ -2,3 +2,24 @@ out
 node_modules
 format.patch
 Makefile
+
+# dependencies
+/.pnp
+.pnp.js
+
+# testing
+/coverage
+
+# production
+build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```

Comment:
Please remoe this red circle image

![image](https://user-images.githubusercontent.com/4616940/133724630-a3ed8c30-f830-4688-9596-701890326753.png)

You need to add an empty line at the end

Reply:
thanks your feedback!